plugins/frontend/backstage-plugin-github-insights/src/components/Widgets/ReadMeCard/ReadmeCard.test.tsx

```diff
@@ -93,19 +93,4 @@ describe('ReadmeCard', () => {
     );
     expect(await screen.findByText(/hello/)).toBeInTheDocument();
   });
-
```

Comment:
This breaks CI in a consistently inconsistent manner. Removing to have stability.